tests/conftest.py

```diff
@@ -0,0 +1,13 @@
+import os
+import pytest
```

Comment:
nit: please add an empty line between both to separate system and global imports.